pkg/util/instanceselector/instanceselector.go

```diff
@@ -470,14 +466,27 @@ func ResourcesToInstanceType(ps *api.PodSpec) (string, *bool, error) {
 		return selector.defaultInstanceType, nil, nil
 	}
 
-	instanceType, needsSustainedCPU := selector.getInstanceFromResources(ps.Resources, ps.InstanceType)
+	instanceType, needsSustainedCPU := selector.getInstanceFromResources(ps.Resources, globInstanceType(ps.InstanceType))
 	if instanceType == "" {
 		msg := "could not compute instance type from Spec.Resources. It's likely that the Pod.Spec.Resources specify an instance that doesnt exist in the cloud"
 		return "", nil, fmt.Errorf(msg)
 	}
 	return instanceType, &needsSustainedCPU, nil
 }
 
+func globInstanceType(instanceType string) func(inst InstanceData) bool {
```

Comment:
I had to stare at this for a while to figure out what's going on. How about calling it `makeInstanceTypeGlobberFunc` or something similar so the reader has an easier time figuring out the logic here?